src/test-resources/generator/generate-axe-cli-resources.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as axe from 'axe-core';
+import * as child_process from 'child_process';
+import * as path from 'path';
+import * as url from 'url';
+
+const axeCoreVersion = (axe as any).version;
+const axeSourcePath = require.resolve('axe-core/axe.min.js');
+const testResourcesDir = path.join(__dirname, '../');
+
+const testUrls: Record<string, string> = {
+    'w3citylights': 'https://www.w3.org/WAI/demos/bad/before/home.html',
+    'basic': url.pathToFileURL(path.join(testResourcesDir, 'basic.html')).toString(),
+};
+
+async function generateResources() {
```

Comment:
I'm not sure if I'm missing something here but: does this function needs to be marked as `async`?

Reply:
Nope! This is a leftover from an earlier implementation, good catch